xonsh/environ.py

```diff
@@ -1217,8 +1215,8 @@ def DEFAULT_VARS():
             is_str_str_dict,
             to_str_str_dict,
             dict_to_str,
-            dict(PTK2_STYLE),
-            "A dictionary containing custom prompt_toolkit style definitions.",
+            {},
```

Comment:
Is there a way to make this a special mapping type that warns the user (using `xonsh.tools.print_warning`) when this is being used? Or maybe add a boolean deprecated attr to `Var` that the `get()` method could use to warn that this variable is deprecated. 

Or maybe there is a deprecation warning elsewhere in the PR?

Reply:
Oh I see it below, where it is being used. nm,